scenarioo-client/app/shared/navigation/applicationInfoPopup.html

```diff
@@ -126,7 +126,7 @@ <h3>Version: {{version.version}}</h3>
             <h3>Build date: {{version.buildDate}}</h3>
             <h3>API version: {{version.apiVersion}}</h3>
             <h3>Internal format version: {{version.aggregatedDataFormatVersion}}</h3>
-            <h3>Documentation: <a href="https://github.com/scenarioo/scenarioo/blob/{{version.releaseBranch}}/docs/README.md" target="_blank">Documentation for Version {{version.version}}</a></h3>
+            <h3>Documentation: <a href="http://www.scenarioo.org/docs/{{version.documentationVersion}}" target="_blank">Documentation for Version {{version.documentationVersion}}</a></h3>
```

Comment:
I would remove the link here, as it does not belong to the "Version" tab. Instead I would rename the "Scenarioo" tab to "Documentation" so that it's clear where to find the link.

Reply:
@adiherzog I dont want to change that right now - no time for it - please open a follow up issue, if you think its important to restructure that dialog - now it is exactly as it was before - out of scope for this story. Thanks.

Reply:
and by the way: I think it is not that bad as it is now - wouldnt remove form the versions tab, since that somehow also is something that belongs to the version, where you find the version's documentation.